### ISSUE TITLE
Adaptive learning rate with Adam, Torch and a Custom Network

### ISSUE THREAD

**Issue by mescarra** [NVIDIA/DIGITS#1807]
![screenshot from 2017-08-31 12-15-21](https://user-images.githubusercontent.com/13679781/29931032-71299c1a-8e46-11e7-8357-a550f4090ba0.png)

Using Torch, a Custom Network and Adam solver, the adaptive learning rate (any of the options) does not work.

At least, what I see during training, on the learning rate graph is a constant learning rate. Can you reproduce this? It this a bug or a non-implemented feature?

Thanks in advance.
